Automattic/jetpack
# Triage: update issue forms and automate label updates

#### Changes proposed in this Pull Request:

GitHub recently updated their issue predefs, and now allow you to create issue forms. Those are interactive, rather than just relying on inline comments in the issue body placeholder, so there is more chance that they'll actually be used.

In this PR, I've done the following:
﻿- Add new GitHub issue form handlers.
- Remove old templates that will be replaced by our new setup.
- Listen to changes to issues in our Gardening action (issues opened or reopened).
- Gardening: add new task to automatically add labels to new issues, based on the standardized content of the new issues.

This should help us better triage new issues in the monorepo, and thus help make sure that plugin authors, regardless of the plugin, can get to their issues faster, without having to rely on manual triage.

#### Jetpack product discussion

* No

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check for typos
* You can check the new forms by viewing the files directly:
    * https://github.com/Automattic/jetpack/blob/add/issue-forms-and-triage/.github/ISSUE_TEMPLATE/bug-report.yml
    * https://github.com/Automattic/jetpack/blob/add/issue-forms-and-triage/.github/ISSUE_TEMPLATE/enhancement.yml
    * https://github.com/Automattic/jetpack/blob/add/issue-forms-and-triage/.github/ISSUE_TEMPLATE/feature-request.yml
* I'm not sure we can check the new task before this gets merged.
